src/popup/pages/Networks/index.js

```diff
@@ -63,16 +63,9 @@ class NetworkPage extends React.Component {
     }
 }
 
-const mapStateToProps = (state) => ({
-    netConfig: state.network,
-});
-
+const mapStateToProps = (state) => ({});
 function mapDispatchToProps(dispatch) {
-    return {
-        updateNetConfigList: (config) => {
-            dispatch(updateNetConfigList(config))
-        },
-    };
```

Comment:
what are these fixes? were these fields unused?

Reply:
Inner updateNetConfigList is not defined, and luckily whole thing was unused

Reply:
ah yeah we just set it once in the constructor